src/quic/node_quic_session.h

```diff
@@ -1460,6 +1493,7 @@ class QuicSession : public AsyncWrap,
   static const ngtcp2_conn_callbacks callbacks[2];
 
   BaseObjectPtr<QuicState> quic_state_;
+  BaseObjectPtr<QLogStream> qlog_stream_;
```

Comment:
So … what if the qlog stream’s `'data'` listener holds back a reference ot the QuicSession? There’s a `qlog C++ stream → qlog JS stream → 'data listener' →JS QuicSession → C++ QuicSession → qlog C++ stream` circular reference that the GC can’t see through, right?

Reply:
Per our slack conversation, I updated this to a `BaseObjectWeakPtr` to break that cycle.